docs/CustomCommands.md

```diff
@@ -6,7 +6,9 @@ title: Custom Commands
 If you want to extend the browser instance with your own set of commands there is a method called `addCommand` available from the browser object. You can write your command in a synchronous (default) way the same way as in your specs or asynchronous (like when using WebdriverIO in standalone mode). The following example shows how to add a new command that returns the current url and title as one result only using synchronous commands:
 
 ```js
-browser.addCommand("getUrlAndTitle", (customVar) => {
+browser.addCommand("getUrlAndTitle", function (customVar) {
+    // this is same as browser, this === browser
+    // and has type WebdriverIO.Browser<void> & WebDriver.Client<void>
```

Comment:
docs should not contain TypeScript code as it might confuse user not knowing typescript.

Reply:
fixed